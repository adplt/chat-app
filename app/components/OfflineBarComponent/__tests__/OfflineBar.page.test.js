import React from 'react';
import renderer from 'react-test-renderer';
import OfflineBar from '../OfflineBar.component';

describe('OfflineBar component', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<OfflineBar />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
