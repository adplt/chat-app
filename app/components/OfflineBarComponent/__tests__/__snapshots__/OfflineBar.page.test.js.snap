// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`OfflineBar component renders correctly 1`] = `null`;
